geopandas/sindex.py

```diff
@@ -243,6 +244,28 @@ def query_bulk(self, geometry, predicate=None, sort=False):
                 input_geometry_index.extend([i] * len(res))
             return np.vstack([input_geometry_index, tree_index])
 
+        def intersection(self, coordinates, objects=False):
+            """Find tree geometries that intersect the input coordinates.
+
+            Parameters
+            ----------
+            coordinates : sequence or array
+                Sequence of the form (min_x, min_y, max_x, max_y)
+                to query a rectangle or (x, y) to query a point.
+            objects : True or False
+                If True, return the label based indexes. If False, integer indexes
+                are returned.
```

Comment:
Do we want to add this to the docs here? It's almost like were publicizing this feature just so we can remove it. I wouldn't remove the documentation that already exists in the pygeos wrapper, but maybe we can not add it for the rtree wrapper...

Reply:
Keep it there for our own reference.

Reply:
Ok